media/com_templates/js/admin-template-toggle-switch.es6.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+function showDiffChanged() {
+  const JformShowDiff = document.getElementById('jform_show_diff');
+  const diffMain = document.getElementById('diff-main');
+
+  setTimeout(() => {
+    if (JformShowDiff.inputs[0] && JformShowDiff.inputs[0].checked === true) {
+      diffMain.style.display = 'none';
```

Comment:
You can define that through CSS.

Reply:
@Anu1601CS What do you mean exactly?

Reply:
@laoneo What do you mean exactly?

Reply:
You can define display none in the CSS file and for the active class a display block. Like that you control in CSS the visibility and not JS.